rdkit/Chem/Descriptors.py

```diff
@@ -181,6 +181,21 @@ def MinAbsPartialCharge(mol, force=False):
 MinAbsPartialCharge.version = "1.0.0"
 
 from rdkit.Chem.EState.EState import MaxEStateIndex, MinEStateIndex, MaxAbsEStateIndex, MinAbsEStateIndex
+from rdkit.Chem.QED import qed
+
+
+def _FingerprintDensity(mol, func, *args, **kwargs):
```

Comment:
Yeah, I agree with you. I think the count-based fingerprints should be using `fp.GetNonzeroElements()` (which returns the number of set "bits") instead of `GetTotalVal()`